api/v1alpha1/common_test.go

```diff
@@ -908,6 +938,34 @@ func TestGetValuesFileDir(t *testing.T) {
 	}
 }
 
+// TestOverrideGetValuesFileDir for Override.GetValueFileDir
+func TestOverrideGetValuesFileDir(t *testing.T) {
```

Comment:
This test should be repeated.

https://github.com/kubebb/core/blob/176c0d80307d57bf2e9c5785f8769644423be293/api/v1alpha1/common_test.go#L880

Reply:
😅 My mistake.